test/Identity.FunctionalTests/ManagementTests.cs

```diff
@@ -193,6 +194,31 @@ public async Task CanRemoveExternalLogin()
             AssertClaimsNotEqual(principals[0], principals[1], "AspNet.Identity.SecurityStamp");
         }
 
+        [Fact]
+        public async Task CanSeeExternalLoginProviderDisplayName()
+        {
+            // Arrange
+            void ConfigureTestServices(IServiceCollection services) => services.SetupTestThirdPartyLogin();
+
+            var server = ServerFactory
+                .WithWebHostBuilder(whb => whb.ConfigureTestServices(ConfigureTestServices));
+
+            var client = server.CreateClient();
+
+            // Act
+            var userName = Guid.NewGuid().ToString();
+            var email = $"{userName}@example.com";
+            var index = await UserStories.RegisterNewUserWithSocialLoginAsync(client, userName, email);
+            var manage = await index.ClickManageLinkWithExternalLoginAsync();
+            var externalLogins = await manage.ClickExternalLoginsAsync();
+
+            // Assert
+            var title = externalLogins.Document.GetElementsByTagName("h4").FirstOrDefault(e => e.TextContent == "Registered Logins");
+            var table = title?.NextElementSibling as IHtmlTableElement;
+            var firstCell = table?.Bodies?.FirstOrDefault()?.Rows.FirstOrDefault()?.Cells?.FirstOrDefault();
+            Assert.Equal("Contoso auth", firstCell?.TextContent);
```

Comment:
@javiercn is this the recommended way to verify this change in the UI tests?

Reply:
Good enough